pkg/rule/api/v1.go

```diff
@@ -128,6 +130,27 @@ func (api *API) alerts(r *http.Request) (interface{}, []error, *qapi.ApiError) {
 	return res, nil, nil
 }
 
+func (api *API) stateAlerts(r *http.Request) (interface{}, []error, *qapi.ApiError) {
```

Comment:
why are we returning `interface{}` here?

Reply:
Why are there `[]error` and `*api.ApiError` here? This needs to return proper data.